src/rsa/padding.rs

```diff
@@ -14,16 +14,19 @@
 
 use {der, digest};
 
+pub trait Encoding {
```

Comment:
We should add a comment about why we use the terms "Encoding" and "Verification" instead of terms that seem more like duals like Encoding/Decoding or Signing/Verification. See my suggested text in my rebase-and-tweak branch.